csharp/src/Ice/CollocatedRequestHandler.cs

```diff
@@ -145,39 +147,38 @@ private bool SentAsync(OutgoingAsyncBase outAsync)
             return true;
         }
 
-        private async ValueTask InvokeAllAsync(Ice.OutputStream os, int requestId)
+        private async ValueTask InvokeAllAsync(OutgoingRequestFrame outgoingRequestFrame, int requestId)
```

Comment:
I suggest simply `request` for the OutgoingRequestFrame parameter.

Reply:
What about the `incomingRequestFrame` created bellow in the method body